challenger/src/multi_field_challenger.rs

```diff
@@ -181,6 +181,14 @@ where
     PF: PrimeField,
     P: CryptographicPermutation<[PF; WIDTH]>,
 {
+    /// The sampled bits are not perfectly uniform, but we can bound the error: every sequence
+    /// appears with probability 1/p-close to uniform (1/2^b).
+    ///
+    /// Proof:
+    /// We denote p = F::ORDER_U64, and b = `bits`.
```

Comment:
One tiny change then good to merge.
```suggestion
    /// We denote p = F::ORDER_U32, and b = `bits`.
```